app/views/mfa_confirmation/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<%# All content on this page is currently set up for MFA user testing.
+ Once ready to ship to prod, translate all text content in Gengo %>
+
+<% title t('titles.mfa_setup.first_authentication_method') %>
```

Comment:
this template was added but I'm not seeing any tests that call it. WDYT of adding a very simple view spec for it?